main/core.js

```diff
@@ -0,0 +1,174 @@
+'use strict'
+
+const { app, BrowserWindow, dialog } = require('electron')
+const { join } = require('node:path')
+const execa = require('execa')
+const wallet = require('./wallet')
+const assert = require('node:assert')
+const fs = require('node:fs/promises')
+const JSONStream = require('jsonstream')
+const Sentry = require('@sentry/node')
+
+/** @typedef {import('./typings').Context} Context */
+/** @typedef {import('./typings').CoreEvent} CoreEvent */
+/** @typedef {import('./typings').Activity} Activity */
+
+const corePath = join(
+  __dirname,
+  '..',
+  'node_modules',
+  '@filecoin-station',
+  'core',
+  'bin',
+  'station.js'
+)
+console.log('Core binary: %s', corePath)
+
+let online = false
+
+async function setup (/** @type {Context} */ ctx) {
+  ctx.saveModuleLogsAs = async () => {
+    const opts = { defaultPath: 'station.txt' }
```

Comment:
All good ideas!

I had initially used `.txt` because I assumed that would be more accessible, but on macOS at least `.log` gets opened in a proper log viewer, so it's even nicer.

I added `-modules` to the name.

I also added a timestamp, I had found the logs overwriting each other annoying too. I decided to use `date.getTime()` instead of the custom format however, as the seems like unnecessary complexity.